README.md

```diff
@@ -11,20 +11,18 @@ Install and run it first.
 
 ### Build
 
-1. Install: git npm nodejs nasm libpng-dev dh-autoreconf
-2. run `npm install -g gulp`
+1. Install the following: `git`, `npm`, `nodejs`, `nasm`, `libpng-dev` and `dh-autoreconf`.
+2. Run as root or administrator `npm install -g gulp`
```

Comment:
Rephrase this since `gulp` has to be installed systemwide as a tool to run the task, This could be put as
`2. Run sudo npm install -g gulp`